README.md

```diff
@@ -13,9 +13,10 @@ git clone git@github.com:bonitoo-io/telegraf-input-oracle.git
 
 __Modify telegraf configuration__
 ```toml
-[[inputs.execd]]
-  command = ["/path/to/telegraf-input-oracle/oracle_metrics.sh"]
-  signal = "none"
+[[inputs.exec]]
+  commands = ["/path/to/telegraf-input-oracle/oracle_metrics.sh"]
+  timeout = "10s"
```

Comment:
Is 10s enough? DB could have a significantly slower response time when in heavy load. Wouldn't be better to set 60s timeout?